Python_Backend/MainRunner.py

```diff
@@ -129,7 +129,25 @@ def convert_image(im):
 async def recv_image(websocket):
 
     async for message in websocket:
-        temp = message.split(",")
+        #take in message as a json and then get the data from it according to its tags
+        try:
+            as_json = json.loads(message)
+        except json.JSONDecodeError as ex:
+            print("There was an error with the JSON data from the frontend")
+        

```

Comment:
Should there be a pass statement here to skip the malformed message? or would we send an error message in this case?

Reply:
Good catch, it has been updated to include a pass